cmd/provider/main.go

```diff
@@ -124,7 +124,7 @@ func main() {
 		log.Info("Alpha feature enabled", "flag", features.EnableAlphaManagementPolicies)
 	}
 
-	kingpin.FatalIfError(metrics.SetupMetrics(), "Cannot setup AWS metrics hook")
+	metrics.SetupMetrics()
```

Comment:
Yes, because I replace method Register to MustRegister inside this func.

`MustRegister works like Register but registers any number of Collectors and panics upon the first registration that causes an error.`

Reply:
This was done as feedback on the original PR, so we should keep it.